_data/master/navbars/getting-started.yml

```diff
@@ -55,12 +55,14 @@ toc:
       path: /getting-started/kubernetes/upgrade/test
     - title: Migrate your data
       path: /getting-started/kubernetes/upgrade/migrate
-    - title: Upgrade components
+    - title: Upgrade Calico
       path: /getting-started/kubernetes/upgrade/upgrade
     - title: Convert your calicoctl manifests
       path: /getting-started/kubernetes/upgrade/convert
-    - title: Troubleshooting
-      path: /getting-started/kubernetes/upgrade/troubleshooting
+    - title: Delete old data
+      path: /getting-started/kubernetes/upgrade/delete
+    - title: Downgrade Calico
```

Comment:
I might call this "Downgrading Calico" so it doesn't feel so much like a step in the process - we don't want people doing this accidentally because they just go to the next step :p

Reply:
I was thinking the same thing, there's something about a gerund in this case that indicates what you suggest. 👍